### ISSUE TITLE
 DAX studio error v2.8.4 vs v2.8.2- export csv file - "Object reference not set to an instance of an object" 

### ISSUE THREAD

**Issue by PeterCaleys** [DaxStudio/DaxStudio#175]
Hi all,

I often export data tables via DAX Studio towards CSV files (Evaluate [TABLE]) . This always used to work, up until the most recent update from DAX Studio.
- In version 2.8.4, it does not work anymore.
- When reinstalling the old version 2.8.2 , the export works again. 

 The error I keep on getting is: "Object reference not set to an instance of an object". 

Any idea why all of a sudden this error is popping up as from DAX Studio version 2.8.4? What could the reason be?

Many thanks,
Peter

**Comment by dgosbell** [comment]
Hi there, thanks for logging this. I added some code to consistently quote string fields in response to another issue, but the tests I ran did not include tables with null values. This error only occurs when fields in the table contain nulls.

I have a preview build of 2.8.5 available [here](https://ci.appveyor.com/api/buildjobs/rg1rfhdxeeajv98i/artifacts/package%2FDaxStudio_2_8_5_setup.exe) which includes a fix for this if you would like to test this
